### PR TITLE
Add pagination to activity viewer

### PR DESCRIPTION
**Pull request by johnsonw**
Add pagination to activity viewer

Signed-off-by: johnsonw <wjohnson@whamcloud.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1695)

### CODE REVIEW

**Comment by johnsonw** [comment]
pagination

**Comment by johnsonw** [comment]
A few notes:
1. Pagination is on both the activity and the logs viewer
2. Next button is gray if there are no more entries
3. Prev button is gray if there are no previous entries
4. Clicking either next or prev will only fetch if a fetch is not in progress 

pagination_full

File: iml-gui/crate/src/components/stratagem/scan_stratagem_modal.rs [hunk 1]
```diff
@@ -59,7 +59,7 @@ pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg, GMsg>)
                 purge_duration: model.purge_duration.value_as_ms(),
             };
 
-            let req = fetch::Request::api_call("run_stratagem")
+            let req = fetch::Request::api_call("run_stratagem", None::<std::collections::HashMap<&str, &str>>)
```
Comment (ip1981) [inline iml-gui/crate/src/components/stratagem/scan_stratagem_modal.rs]: Maybe add a new method `api_call_foo`?

Comment (johnsonw) [reply iml-gui/crate/src/components/stratagem/scan_stratagem_modal.rs]: Created `api_query`

**Pull request merged**